utils/createClassRegistration.ts

```diff
@@ -0,0 +1,33 @@
+import { ParentRegistrationInput } from "@models/Enroll";
+import { Student } from "@prisma/client";
+import { mutate } from "swr";
+
+/**
+ * createClassRegistration creates a registration for a class given a student
+ * @param student student's record to create
+ * @param classId id of class to create the record to
+ * @returns deleted records
+ */
+export async function createClassRegistration(
+    student: Student,
+    classId: number,
+) {
```

Comment:
Let's be explicit here with types
```suggestion
): Promise<any> {
```